.github/workflows/check-manifest.yml

```diff
@@ -6,6 +6,9 @@ on:
 
 jobs:
   check:
+    # Don't run on forks
+    if: github.repo_owner == 'scikit-learn'
```

Comment:
To be consistent with the other workflows that use a conditional to not run on forks:

```suggestion
    if: github.repository == 'scikit-learn/scikit-learn'
```